### PR TITLE
Alex Yang's code!

### PR DESCRIPTION
**Pull request by yanga9345**
This is it! In all it's glory. Witness my supreme coding ability mortal, and have fear in your heart! For my program can only do one calculation at a time!

### CODE REVIEW

**Comment by yanga9345** [comment]
Also this is Alex Yang

File: ex02.exs [hunk 1]
```diff
@@ -82,23 +83,29 @@ defmodule Ex02 do
...
   end
 
   def next_value(agent) do
     # « your code goes here»
+    Agent.get_and_update(agent, fn total -> {total, total+1} end)
   end
 
-  @global_name :my_global_agent
+  #I hope you like this textArt
+  @global_name :JamesBond #(James Bond) (° ͜ʖ°)=ε/̵͇̿̿/’̿’̿ ̿      ̿ ̿’̿’/̵͇̿̿/ε(o_o) (surprised gunman)
```
Comment (pragdave) [inline ex02.exs:96]: JamesBond. Agent. I get it...

# +1

File: ex03.exs [hunk 2]
```diff
@@ -46,25 +51,61 @@ defmodule Ex03 do
...
   35 points:
      it works and passes all tests:    25
-     it contains no conditional logic:  3
+     it contains no conditional logic:  3       WOOOOO!!!! I got 3 points at least!!!!
      it is nicely structured            7
   """
 
+  def split(collection, process_count) do
+    tempList = Enum.to_list(collection) #converts collection to a list
+    num = div(List.last(tempList), process_count) + 1 #num = the size of each of the sublists(add one in case uneven split)
```
Comment (pragdave) [inline ex03.exs:60]: Actually the +1 fails when the division is exact. It should be `div(Enum.count(coll) + process_count - 1, process_count)`

File: ex03.exs [hunk 3]
```diff
@@ -46,25 +51,61 @@ defmodule Ex03 do
 
   35 points:
      it works and passes all tests:    25
-     it contains no conditional logic:  3
+     it contains no conditional logic:  3       WOOOOO!!!! I got 3 points at least!!!!
```
Comment (pragdave) [inline ex03.exs:54]: Indeed you do:

# -32

**Comment by pragdave** [comment]
Fear in my heart indeed.

**Pull request closed without merging**